### PR TITLE
Watering-Can and Chest Component Tests

### PR DESCRIPTION
**Pull request by LiamAnd**
Helps with #544

### CODE REVIEW

File: source/core/src/main/com/csse3200/game/components/inventory/InventoryDisplay.java [hunk 1]
```diff
@@ -28,7 +28,7 @@
  */
 public class InventoryDisplay extends UIComponent {
 	private InventoryComponent inventory;
-	private final Skin skin = ServiceLocator.getResourceService().getAsset("gardens-of-the-galaxy/gardens-of-the-galaxy.json", Skin.class);
+	//private final Skin skin = ServiceLocator.getResourceService().getAsset("gardens-of-the-galaxy/gardens-of-the-galaxy.json", Skin.class);
```
Comment (JTrenerry) [inline source/core/src/main/com/csse3200/game/components/inventory/InventoryDisplay.java:31]: I don't think this should be removed.

File: source/core/src/test/com/csse3200/game/components/placeables/ChestComponentTest.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        chest.addComponent(new InventoryComponent())
+        .addComponent(new ChestComponent());
+        //.addComponent(new InventoryDisplay("refreshChest", "toggleChest", 30, 10, false))
+
+        //InventoryDisplay chestInv = chest.getComponent(InventoryDisplay.class);
+
+        //assertFalse(chestInv.isOpen());     //chest is not opened
+            //trigger the open chest action
+        //chest.getEvents().trigger("interact");
+        //assertTrue(chestInv.isOpen());      //chest is open
```
Comment (JTrenerry) [inline source/core/src/test/com/csse3200/game/components/placeables/ChestComponentTest.java:31]: should be removed.

**Review by JTrenerry — DISMISSED** [review]
Left comments

File: source/core/src/main/com/csse3200/game/components/inventory/InventoryDisplay.java (hunk 1 above)
Comment (LiamAnd) [reply source/core/src/main/com/csse3200/game/components/inventory/InventoryDisplay.java:31]: whoops missed that

**Review by JTrenerry — APPROVED** [review]

**Pull request merged**